staging/src/k8s.io/api/testdata/HEAD/apps.v1.DaemonSet.json

```diff
@@ -1575,40 +1575,41 @@
                   "operator": "Exists"
                 }
               ]
-            }
+            },
+            "minDomains": 1680820399
```

Comment:
FYI, I'm reworking how these fixtures are generated in https://github.com/kubernetes/kubernetes/pull/108219 so that a tiny change like this PR would not ripple into huge changes in these fixtures. After my fix, this PR would result in an update like https://github.com/liggitt/kubernetes/commit/a78331610dd94ef9bac5f8f4ab8a2d0b6a820851

Reply:
https://github.com/kubernetes/kubernetes/pull/108219 is now merged

The easiest way to rebase on master and rerun hack/update-generated-api-compatibility-data.sh would probably be something like:

```
# Revert changes to HEAD fixture files
git checkout d5263fe -- staging/src/k8s.io/api/testdata

# Squash to a single commit
git reset --soft d5263fe
git commit -m "Add MinDomains API to TopologySpreadConstraints field"

# Rebase on master (assumes your remote for Kubernetes is named "upstream")
git fetch upstream
git rebase upstream/master

# Regenerate fixture files
hack/update-generated-api-compatibility-data.sh
git add staging/src/k8s.io/api/testdata
git commit -m "Generated files"
```

Reply:
@liggitt Thanks for sharing. I'll try that.